plugins/module_utils/redfish_utils.py

```diff
@@ -1924,7 +1924,14 @@ def set_bios_attributes(self, attributes):
         # Get the SettingsObject URI to apply the attributes
         set_bios_attr_uri = data.get("@Redfish.Settings", {}).get("SettingsObject", {}).get("@odata.id")
         if set_bios_attr_uri is None:
-            return {'ret': False, 'msg': "Settings resource for BIOS attributes not found."}
+            # WORKAROUND
+            # Some Gigabyte systems do not implement @Redfish.Settings, but
+            # implement the settings resource.
+            vendor = self._get_vendor()['Vendor']
+            if vendor == 'Gigabyte':
```

Comment:
I'm hoping the vendor string from the service root comes back like this. @jyundt or @sseekamp can you confirm this? Does Gigabyte use a different value?

Reply:
Actually it's:
`"Vendor": "AMI"`

Reply:
Ah, so this actually impacts more than just Gigabyte systems... Anything with AMI as the firmware supplier is impacted... Thanks for confirming!

Reply:
Thanks for working this patch up for us - much appreciated!

Reply:
Updated the change to look for AMI; would you be able to test this change to confirm it fixes the issue?